cpp/proto/arcticc/pb2/azure_storage.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package arcticc.pb2.azure_storage_pb2;
+
+message Config {
+    string container_name = 1;
+    string credential_name = 2;
+    string credential_key = 3;
+    string endpoint = 4;
+    uint32 max_connections = 5;
```

Comment:
I know this is copied from S3, but I wonder if it's correct to have it in the permanent storage config.

Reply:
I would say let's stick to S3 as Azure Storage support should be like a drop-in replacement for S3 here.